--- ipmiutil-2.1.2/util/Makefile.am.orig	2008-06-24 23:11:30.000000000 +0200
+++ ipmiutil-2.1.2/util/Makefile.am	2008-07-20 15:04:45.542041412 +0200
@@ -48,32 +48,21 @@
 LDADD += -lpthread
 
 sbin_PROGRAMS = ipmiutil idiscover 
-EXTRA_PROGRAMS = events ipmi_port libipmiutil.a 
+EXTRA_PROGRAMS = events ipmi_port
+noinst_LTLIBRARIES = libipmiutil.la libimbapi.la
 OLDPROGS = alarms bmchealth fruconfig getevent hwreset icmd isolconsole pefconfig sensor showsel tmconfig wdt
 # xmlconfig dropped
 
 # To build an imb api library (libimbapi.a):
-libimbapi.a:	imbapi.c
-	$(CC) -DHAVE_CONFIG_H -I. -I.. -DLINUX -fPIC $(CFLAGS) -c imbapi.c
-	$(AR) cru libimbapi.a imbapi.o
-	$(RANLIB) libimbapi.a
-	rm -f imbapi.o
+libimbapi_la_SOURCES = imbapi.c
+libimbapi_la_CFLAGS = -DHAVE_CONFIG_H -I. -I.. -DLINUX -fPIC -c
  
 # To build a lib to support all of the IPMI drivers (libipmiutil.a)
-libipmiutil.a:	$(CMDMOD:.c=.o) @LANPLUS_LIB@
-	$(AR) cru libipmiutil.a $(CMDMOD:.c=.o) @LANPLUS_LIB@
-	$(RANLIB) libipmiutil.a
+libipmiutil_la_SOURCES = $(CMDMOD)
+libipmiutil_la_LIBADD = @LANPLUS_LIB@
+libipmiutil_la_CFLAGS = $(AM_CFLAGS)
 
 # To build ipmiutil, need to use METACFLAGS for each .c/.o
-
-idiscover:	idiscover.c
-	$(CC) $(CFLAGS) $(LDFLAGS) -o idiscover idiscover.c
-
-events:         events.c
-	$(CC) $(CFLAGS) $(LDFLAGS) -DTEST -o events events.c
-
-ipmi_port:      ipmi_port.c
-	$(CC) $(CFLAGS) $(LDFLAGS) -o ipmi_port ipmi_port.c 
 
 ipmiutil_SOURCES  = $(METASOURCE) 
 
