--- ipmiutil-2.2.0/util/Makefile.am.orig	2008-08-28 00:43:57.162120901 +0200
+++ ipmiutil-2.2.0/util/Makefile.am	2008-08-28 00:44:44.674119225 +0200
@@ -71,8 +71,10 @@
 #	$(CC) $(CFLAGS) $(LDFLAGS) -o ipmiutil $(METASOURCE:.c=.o)
 
 ievents_SOURCES    = ievents.c
+ievents_CPPFLAGS = $(AM_CPPFLAGS) -UMETACOMMAND -DTEST
 
 idiscover_SOURCES = idiscover.c 
+idiscover_CPPFLAGS = $(AM_CPPFLAGS) -UMETACOMMAND
 
 ipmi_port_SOURCES = ipmi_port.c 
 
@@ -82,9 +84,5 @@
 
 all-am: Makefile $(sbin_PROGRAMS) $(EXTRA_PROGRAMS)
 
-install-data-am:  $(EXTRA_PROGRAMS)
-	mkdir -p $(DESTDIR)$(extradir)
-	cd $(DESTDIR)/$(sbindir); strip $(sbin_PROGRAMS)
-
 clean-generic:   
 	rm -f $(EXTRA_PROGRAMS) $(OLDPROGS) $(TESTPROGS)
