--- ipmiutil-2.1.2/util/Makefile.am.orig	2008-07-20 15:37:30.106034707 +0200
+++ ipmiutil-2.1.2/util/Makefile.am	2008-07-20 15:44:38.450034707 +0200
@@ -47,8 +47,8 @@
 METASOURCE = ipmiutil.c alarms.c bmchealth.c events.c fruconfig.c getevent.c hwreset.c icmd.c pefconfig.c sensor.c showsel.c tmconfig.c wdt.c isolconsole.c idiscover.c $(CMDMOD) 
 LDADD += -lpthread
 
-sbin_PROGRAMS = ipmiutil idiscover 
-EXTRA_PROGRAMS = events ipmi_port
+sbin_PROGRAMS = ipmiutil idiscover events
+EXTRA_PROGRAMS = ipmi_port
 noinst_LTLIBRARIES = libipmiutil.la libimbapi.la
 OLDPROGS = alarms bmchealth fruconfig getevent hwreset icmd isolconsole pefconfig sensor showsel tmconfig wdt
 # xmlconfig dropped
@@ -67,8 +67,10 @@
 ipmiutil_SOURCES  = $(METASOURCE) 
 
 events_SOURCES    = events.c
+events_CPPFLAGS = $(AM_CPPFLAGS) -UMETACOMMAND -DTEST
 
 idiscover_SOURCES = idiscover.c 
+idiscover_CPPFLAGS = $(AM_CPPFLAGS) -UMETACOMMAND
 
 ipmi_port_SOURCES = ipmi_port.c 
 
@@ -78,9 +80,5 @@
 
 all-am: Makefile $(sbin_PROGRAMS) $(EXTRA_PROGRAMS)
 
-install-data-am:  $(EXTRA_PROGRAMS)
-	$(INSTALLBIN) events    $(DESTDIR)/$(datadir)/ipmiutil
-	$(INSTALLBIN) ipmi_port $(DESTDIR)/$(datadir)/ipmiutil
-
 clean-generic:   
 	rm -f $(EXTRA_PROGRAMS) $(OLDPROGS)
