diff -ruNp ipmiutil-2.9.7.orig/util/idiscover.c ipmiutil-2.9.7/util/idiscover.c
--- ipmiutil-2.9.7.orig/util/idiscover.c	2015-10-12 19:13:02.000000000 +0200
+++ ipmiutil-2.9.7/util/idiscover.c	2015-11-29 01:24:27.639654074 +0100
@@ -525,7 +525,7 @@ int sock_init( char *_interface, char *_
       }
       if (n < 0) rv = LAN_ERR_OTHER; /*-13*/
    } else { /* valid _interface string */
-      if (strchr(_interface, '.') != NULL)
+      if (0)
         {   /* assume it is an IP address*/
           if ((rv = inet_pton(AF_INET, _interface, &_srcaddr.sin_addr)) < 0)
             printerr("inet_pton: %s\n", showlasterr());
