--- ipmiutil-1.5.8/util/Makefile.am~	2004-11-10 17:49:51.000000000 +0200
+++ ipmiutil-1.5.8/util/Makefile.am	2005-03-02 13:23:02.000000000 +0200
@@ -3,7 +3,7 @@
 AM_CPPFLAGS = -DLINUX -DLOCALEDIR=\"$(localedir)\"
 INCLUDES = -I..
 CMDMOD = ipmicmd.c imbapi.c ipmimv.c ipmignu.c
-LDFLAGS = -lfreeipmi -L../freeipmi
+LDFLAGS = -lfreeipmi
 # To build without gnu freeipmi, uncomment the following two lines.
 # LDFLAGS = 
 # CMDMOD = ipmicmd.c imbapi.c ipmimv.c ipmignustub.c
