diff -ruN ipmiutil-2.9.7.orig/configure.ac ipmiutil-2.9.7/configure.ac
--- ipmiutil-2.9.7.orig/configure.ac	2015-10-12 19:13:02.000000000 +0200
+++ ipmiutil-2.9.7/configure.ac	2015-11-29 01:21:44.570406534 +0100
@@ -2,7 +2,7 @@
 AC_INIT(ipmiutil,2.9.7)
 AC_CONFIG_SRCDIR(util/ipmiutil.c)
 echo "ipmiutil version ${PACKAGE_VERSION}"
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 AC_PREFIX_DEFAULT(/usr)
 AM_INIT_AUTOMAKE()
 
