--- ipmiutil-3.1.2/util/Makefile.am.orig	2018-07-24 20:39:41.644112439 +0200
+++ ipmiutil-3.1.2/util/Makefile.am	2018-07-24 20:43:01.740776823 +0200
@@ -114,9 +114,9 @@
 $(SHR_LIB):	$(CMDMOD:.c=.o) @LANPLUS_LIB@ @LIBSENSORS@ $(TMPOBJL) $(TMPOBJA) 
 	if [ "x@LANPLUS_SAM@" = "xyes" ]; then \
 	   ar x @LANPLUS_LIB@ ; \
-	   $(CC) $(LDFLAGS) -shared $(LDNAME) -o $(SHR_LIB) $(LIBOBJ) $(TMPOBJL) $(LANPLUS_OBJ); \
+	   $(CC) $(LDFLAGS) -shared $(LDNAME) -o $(SHR_LIB) $(LIBOBJ) $(TMPOBJL) $(LANPLUS_OBJ) -lcrypto; \
 	else \
-	   $(CC) $(LDFLAGS) -shared $(LDNAME) -o $(SHR_LIB) $(LIBOBJ) $(TMPOBJA) ; \
+	   $(CC) $(LDFLAGS) -shared $(LDNAME) -o $(SHR_LIB) $(LIBOBJ) $(TMPOBJA) -lcrypto; \
 	fi 
 
 $(SHR_BLIB):	$(SHR_LIB)
