--- ipmiutil-2.7.8/util/Makefile.am.orig	2011-09-13 23:34:36.000000000 +0200
+++ ipmiutil-2.7.8/util/Makefile.am	2011-09-17 15:06:56.278234517 +0200
@@ -64,75 +64,47 @@
 # EXTRA_PROGRAMS are built by default but not included in the install package
 # TESTPROGS are not built by default.  To build, do 'make ifruset', for example.
 sbin_PROGRAMS = ipmiutil idiscover ipmi_port ievents
-EXTRA_PROGRAMS = libipmiutil.a ipmi_sample
+noinst_LTLIBRARIES = libipmiutil.la libimbapi.la
 TESTPROGS = libimbapi.a iconfig ipmimv ifruset
 # OLDPROGS are old/previous binaries that may exist and need to be deleted.
 OLDPROGS = alarms bmchealth fruconfig getevent hwreset icmd isolconsole pefconfig sensor showsel tmconfig wdt
 
 # To build an imb api library (libimbapi.a):
-libimbapi.a:	imbapi.c
-	$(CC) -I. -I.. @OS_CFLAGS@ -fPIC $(CFLAGS) -o $(tmpdir)/imbapi.o -c imbapi.c
-	$(AR) cru libimbapi.a $(tmpdir)/imbapi.o
-	$(RANLIB) libimbapi.a
-	rm -f $(tmpdir)/imbapi.o
+libimbapi_la_SOURCES = imbapi.c
+libimbapi_la_CFLAGS = -I. -I.. @OS_CFLAGS@ -fPIC
  
 # To build a lib to support all of the IPMI drivers (libipmiutil.a)
-libipmiutil.a:	$(CMDMOD:.c=.o) @LANPLUS_LIB@
-	if [ -f "@LANPLUS_LIB@" ]; then \
-	    $(CC) $(CFLAGS_SAM) -c ipmilanplus.c; \
-	    $(CC) $(CFLAGS_SAM) -c ipmilan.c; \
-	    ar x @LANPLUS_LIB@ ; fi
-	$(AR) cru libipmiutil.a $(CMDMOD:.c=.o) $(LANPLUS_OBJ) 
-	$(RANLIB) libipmiutil.a
-	if [ -f "@LANPLUS_LIB@" ]; then \
-	    rm -f $(LANPLUS_OBJ) ipmilan.o ipmilanplus.o; fi 
-
-# To build ipmiutil, need to use METACFLAGS for each .c/.o
-
-idiscover:	idiscover.c
-	$(CC) @OS_CFLAGS@ $(CFLAGS) $(LDFLAGS) $(OS_LF) -o idiscover idiscover.c
-
-ipmimv: 	ipmimv.c
-	$(CC) @OS_CFLAGS@ $(CFLAGS) $(LDFLAGS) -DTEST_BIN -o ipmimv ipmimv.c
-
-ievents:         ievents.c
-	$(CC) $(CFLAGS) $(LDFLAGS) -DALONE -o ievents ievents.c 
-
-ipmi_sample:	libipmiutil.a ipmi_sample.c
-	$(CC) $(CFLAGS_SAM) -c ipmi_sample.c
-	$(CC)  -g -O2 -o ipmi_sample  ipmi_sample.o libipmiutil.a $(LDFLAGS) $(LDADD)
-
-ifruset:	libipmiutil.a ifruset.c ifru_picmg.c
-	$(CC) $(CFLAGS_SAM) -c ifruset.c
-	$(CC) $(CFLAGS_SAM) -c ifru_picmg.c
-	$(CC)  -g -O2 -o ifruset ifruset.o ifru_picmg.o libipmiutil.a $(LDFLAGS) $(LDADD)
-
-ipmi_port:      ipmi_port.c
-	$(CC) $(CFLAGS) $(LDFLAGS) -o ipmi_port ipmi_port.c 
-
-#xmlconfig:      xmlconfig.c libipmiutil.a
-#	$(CC) $(CFLAGS) $(LDFLAGS) -o xmlconfig xmlconfig.c libipmiutil.a @LANPLUS_CRYPTO@
-
-iconfig:      iconfig.c libipmiutil.a
-	$(CC) $(CFLAGS) $(LDFLAGS) -o iconfig iconfig.c libipmiutil.a @LANPLUS_CRYPTO@
+libipmiutil_la_SOURCES = $(CMDMOD) isol.c
+libipmiutil_la_LIBADD = @LANPLUS_LIB@ -lcrypto
+libipmiutil_la_CFLAGS = $(AM_CFLAGS)
 
 ipmiutil_SOURCES  = $(METASOURCE) 
-# ipmiutil:	$(METASOURCE:.c=.o) 
-#	$(CC) $(CFLAGS) $(LDFLAGS) -o ipmiutil $(METASOURCE:.c=.o)
 
 ievents_SOURCES    = ievents.c
+ievents_CPPFLAGS = $(AM_CPPFLAGS) -UMETACOMMAND -DALONE
 
 idiscover_SOURCES = idiscover.c 
+idiscover_CPPFLAGS = $(AM_CPPFLAGS) -UMETACOMMAND @OS_CFLAGS@
+
+ipmimv_SOURCES = ipmimv.c
+ipmimv_CPPFLAGS = $(AM_CPPFLAGS) -DTEST_BIN
 
+ifruset_SOURCES = ifruset.c ifru_picmg.c
+ifruset_LDADD = libipmiutil.la
+
+ipmi_sample_SOURCES = ipmi_sample.c
+ipmi_sample_LDADD = libipmiutil.la
+ 
 ipmi_port_SOURCES = ipmi_port.c 
 
 # xmlconfig_SOURCES = xmlconfig.c $(CMDMOD) 
 
+iconfig_SOURCES = iconfig.c
+iconfig_LDADD = libipmiutil.la @LANPLUS_CRYPTO@
+ 
 EXTRA_DIST = imb_api.h ipmicmd.h ipmidir.h ipmi_ioctls.h ipmilan.h ipmilanplus.h AnsiTerm.h iekanalyzer.h ifirewall.h ifwum.h ihpm.h ipicmg.h ipmiutil.h ipmiva.h md2.h oem_dell.h oem_fujitsu.h oem_intel.h oem_kontron.h oem_sun.h
 
-all-am: Makefile $(sbin_PROGRAMS) $(EXTRA_PROGRAMS)
-
-install-data-am:  $(EXTRA_PROGRAMS)
+install-data-am:
 	mkdir -p $(DESTDIR)$(extradir)
 
 clean-generic:   
